### PR TITLE
use a separate slave lag table to get timestamps

### PR DESCRIPTION
**Pull request by BrianIp**

### CODE REVIEW

**Comment by BrianIp** [comment]
r @michaelfinch @jcesario

File: mysql/dbstat/dbstat.go [hunk 1]
```diff
@@ -349,7 +392,6 @@ func (s *MysqlStatDBs) GetSlaveStats() {
 			s.Db.Log(err)
 		} else {
 			if numBackups > 0 {
-				s.Metrics.SlaveSecondsBehindMaster.Set(float64(-1))
```
Comment (michaelfinch) [inline mysql/dbstat/dbstat.go]: why?

Comment (BrianIp) [inline mysql/dbstat/dbstat.go]: good catch, didn't mean to take out that line there

**Comment by michaelfinch** [comment]
lgtm

**Pull request merged**